.github/workflows/test.yml

```
@@ -0,0 +1,20 @@
+name: Run unit tests
+
+on:
+  push:
```

I noticed tests seem to run twice on pull requests that are made within this project, which I think can be solved by changing the push event to only be for the master branch:

```suggestion
  push:
    branches:
      - master
```

https://github.community/t/duplicate-checks-on-push-and-pull-request-simultaneous-event/18012/2